### PR TITLE
Docs: Update Configuration in Getting Started

### PR DESCRIPTION
**Pull request by khawarlatifkhan**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
Added a comma in the Configuration section.

#### Is there anything you'd like reviewers to focus on?
This is a test PR before working on Configuration Documentation for GSOD.

### CODE REVIEW

**Comment by khawarlatifkhan** [comment]
@kaicataldo So, I created this PR now. I'm guessing the commit-message was according to the guidelines since it doesn't give any errors. I still don't know how to edit a message after I've created the PR.

File: docs/developer-guide/development-environment.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ESLint has a very lightweight development environment that makes updating code f
 
 Go to <https://nodejs.org/> to download and install the latest stable version for your operating system.
 
-Most of the installers come with [npm](https://www.npmjs.com/) already installed, but if for some reason it doesn't work on your system, you can install it manually using the instructions on the site.
+Most of the installers already come with [npm](https://www.npmjs.com/) but if for some reason npm doesn't work on your system, you can install it manually using the instructions on the site.
```
Comment (kaicataldo) [inline docs/developer-guide/development-environment.md:9]:
```suggestion
Most of the installers already come with [npm](https://www.npmjs.com/). If for some reason npm doesn't work on your system, you can install it manually using the instructions on the site.
```

**Comment by kaicataldo** [comment]
Did you read the docs [here](https://github.com/eslint/eslint-github-bot/blob/master/docs/commit-message-check.md#how-do-i-fix-it)? If so, do you mind adding a bit more info about what is confusing? Maybe we can make those docs better!

**Comment by khawarlatifkhan** [comment]
I did, yes.
So, my confusion: The [document](https://github.com/eslint/eslint-github-bot/blob/master/docs/commit-message-check.md#how-do-i-fix-it) mentions 2 commands which can be used to edit the commit message. Since I was using via my browser, I wasn't sure how to edit the commit message from here. But I'm guessing that these commands refer to the "Terminal" on my machine and that is how I can edit the commit message. Is that so? If yes, can I try it with another PR where I deliberately enter the wrong commit message first and then correct it using the Terminal?
I'm sorry if it too confusing and weird. Trying to understand it all.

**Comment by kaicataldo** [comment]
No need to apologize! Learning this stuff takes time, and you're doing great.

> But I'm guessing that these commands refer to the "Terminal" on my machine and that is how I can edit the commit message. Is that so? 

That's right. I don't think there's a way to edit commit messages via GitHub's browser UI.

> If yes, can I try it with another PR where I deliberately enter the wrong commit message first and then correct it using the Terminal?

Absolutely. If it's helpful, we can set aside some time to sit down and go through it together, if you'd like!

**Comment by kaicataldo** [comment]
Looks like we have one comment to resolve and then we can merge this!

**Comment by khawarlatifkhan** [comment]
> Absolutely. If it's helpful, we can set aside some time to sit down and go through it together, if you'd like!
This sounds great, yes. Please let me know when can we do that and I'll pick a small thing to work on when we talk.

**Review by kaicataldo — APPROVED** [review]
LGTM, thank you!

**Review by btmills — APPROVED** [review]
LGTM. Welcome, @khawarlatifkhan! :wave:

**Comment by khawarlatifkhan** [comment]
Thank you @btmills

**Pull request closed without merging**